js/components/typeahead.js

```diff
@@ -73,32 +73,6 @@ Fliplet.FormBuilder.field('typeahead', {
     return rules;
   },
   created: function() {
-    var $vm = this;
```

Comment:
Why is all this deleted?

Reply:
this is an unnecessary part of code. The field options are not rendered here, which could be a reason for losing the value.